bitnami/contour/templates/contour/deployment.yaml

```diff
@@ -110,8 +110,13 @@ spec:
             {{- if .Values.contour.ingressStatusAddress }}
             - --ingress-status-address={{ .Values.contour.ingressStatusAddress }}
             {{- else }}
+            {{- if .Values.envoy.service.multiAz.enabled }}
             - --envoy-service-namespace={{ default .Release.Namespace .Values.contour.envoyServiceNamespace }}
```

Comment:
As this parameter gets the same value, you can move it out of the if/else block

Reply:
Ah! yes... thanks!